gourmet/gtk_extras/LinkedTextView.py

```diff
@@ -132,26 +133,30 @@ def event_after(self, text_view, event):
         self.follow_if_link(text_view, iter)
         return False
 
-    # Looks at all tags covering the position (x, y) in the text view,
-    # and if one of them is a link, change the cursor to the "hands" cursor
-    # typically used by web browsers.
-    def set_cursor_if_appropriate(self, text_view, x, y):
-        hovering = False
-        _, iter_ = text_view.get_iter_at_location(x, y)
-        tags = iter_.get_tags()
-
+    def set_cursor_if_appropriate(self,
+                                  text_view: 'LinkedTextView',
+                                  x: int,
+                                  y: int) -> None:
+        """Set the mouse cursor to be a hand when hovering over a time link.
+
+        Check each tag covering the position (x, y) within the text view.
+        If the text happens to be coloured, which only time links are allowed
+        to be in this application, then set the cursor to a hand.
+        If leaving the area of a time link, set the cursor to an I-beam.
+        """
+        _, itr = text_view.get_iter_at_location(x, y)
+        tags = itr.get_tags()
+
+        hovering_over_link = False
         for tag in tags:
-            href = tag.get_data("href")
-            if href:
-                hovering = True
+            if isinstance(tag.get_property('foreground-gdk'), Gtk.Color):
+                hovering_over_link = True
                 break
-        if hovering != self.hovering_over_link:
-            self.hovering_over_link = hovering
 
-        if self.hovering_over_link:
-            text_view.get_window(Gtk.TextWindowType.TEXT).set_cursor(self.hand_cursor)
-        else:
-            text_view.get_window(Gtk.TextWindowType.TEXT).set_cursor(self.regular_cursor)
+        cursor = self.text_cursor
+        if hovering_over_link:
+            cursor = self.hand_cursor
```

Comment:
sometimes I feel `cursor = self.hand_cursor if hovering_over_link else self.text_cursor` is more concise as you don't have to mentally parse whether the `if` does anything intelligent or noteworthy. It is clear that it's just about the state of `cursor`.

But that's just taste and if you're happy with this PR, please merge it.